scripts/scripts.js

```diff
@@ -136,6 +144,7 @@ function addBackToTop(main) {
  * loads everything that doesn't need to be delayed.
  */
 async function loadLazy(doc) {
+  await initPartytown();
```

Comment:
`initPartytown` function doesn't need to be `async` and we don't need an `await` here

Reply:
removing it